CHANGELOG.md

```diff
@@ -9,6 +9,16 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 The following changes have been implemented but not released yet:
 
+### New Feature
+
+#### node
+
+- It is now possible to prevent a `Session` to self-refresh in NodeJS. To do so, a new
+  parameter is added to the constructor: `Session({ keepAlive: false })`. This prevents
+  the `Session` setting a callback to refresh the Access Token before it expires, which
+  could cause a memory leak in the case of a server-side application with many users, in
+  addition to unnecessary requests sent to the OpenID Provider.
```

Comment:
```suggestion
- It is now possible to prevent a `Session` self-refreshing in NodeJS. To do so, a new
  parameter is added to the constructor: `Session({ keepAlive: false })`. This prevents
  the `Session` setting a callback to refresh the Access Token before it expires, which
  could cause a memory leak in the case of a server-side application with many users.
  It also avoids unnecessary requests being sent to the OpenID Provider.
```